config/config.go

```diff
@@ -150,6 +150,10 @@ func (c *Config) ExtractServiceTLSConfig() (*tls.Config, error) {
 			copy(caFileBytes, defaultProdCA)
 		}
 	} else {
+		_, ok := os.LookupEnv("INTEGRATION_TEST")
+		if !ok {
+			return nil, errors.New("cannot set custom CA outside integration tests")
+		}
```

Comment:
In the case where we do have `INTEGRATION_TEST` and use the custom CA file, should we log it (with the path to the file) to make it easy to debug later for someone who would have forgotten that they have this variable defined in their environment?

Reply:
updated to add a logger